src/adjtrans.jl

```diff
@@ -382,6 +382,20 @@ parent(A::AdjOrTrans) = A.parent
 vec(v::TransposeAbsVec{<:Number}) = parent(v)
 vec(v::AdjointAbsVec{<:Real}) = parent(v)
 
+Base.reshape(v::TransposeAbsVec{<:Number}, ::Val{1}) = parent(v)
+Base.reshape(v::AdjointAbsVec{<:Real}, ::Val{1}) = parent(v)
+
+ # these make eachrow(A') produce simpler views
+@inline Base.unsafe_view(A::Transpose{<:Number, <:AbstractMatrix}, i::Integer, j::AbstractArray) =
+    Base.unsafe_view(parent(A), j, i)
```

Comment:
These should probably be `view`, as the parent might have a specialized `view` implementation that avoids the `SubArray` wrapper.

Reply:
Can't we also do something similar for transposes of `AbstractVector`s, with one `AbstractVector` index?

Reply:
Probably. But that's a somewhat odd thing, and this PR doesn't try to cover all possible views -- just the ones which show up in `eachrow(A')`.

Reply:
Probably. But that's a somewhat odd thing, and this PR doesn't try to cover all possible views.